### ISSUE TITLE
Bug with circles

### ISSUE THREAD

**Issue by BenjaminRueth** [ChairOfStructuralMechanicsTUM/Mechanics_Apps#9]
In a couple of apps we observe a bug in the visualization of circles: The circles centre is at the wrong position and the radius is half the size. 

We can observe the error in the following apps:

* Drehscheibe-Corioliskraft
* Rollercoaster

The following way the error is produced:

* Open Mechanic_Apps.exe
* Open e.g. Corioliskraft, all circles look fine
* Open Diffraction (you have to open THIS app!)
* Open e.g. Corioliskraft again, circles look weird.

**Comment by BenjaminRueth** [comment]
Easier way for testing: bokeh serve Diffraction/ Coriolis_force/

**Comment by BenjaminRueth** [comment]
commenting out line 17
``` from surface3d import Surface3d ```
and replacing line 272 
``` surface = Surface3d(x="x", y="y", z="z", color="color", data_source=source_surf) ```
with dummy code
``` surface = Figure() ```
prevents Diffraction from breaking the circles in Coriolis force.

Actually: Already the import in line 17 on its own break the circles.

**Comment by BenjaminRueth** [comment]
The bug is caused by the library ```vis.js``` that is used by ```surface3d.py```. We can easily produce the error by importing
```
from bokeh.core.properties import Any, Dict, Instance, String
from bokeh.models import ColumnDataSource, LayoutDOM

class Tester(LayoutDOM):
    __javascript__ = ["Diffraction/static/js/vis.min.js"] ## offline resource
```

At this point we can probably not do anything about the bug.

A workaround: Host Diffraction on a different server.

```
bokeh serve --port 5006 Coriolis_force/
bokeh serve --port 5007 Diffraction/
```

Here Coriolis_force is not influenced by Diffraction anymore.

**BUT** this requires special treatment of Diffraction on all layers of our application.

**Comment by FrancescaTaddei** [comment]
Thanks! Let us use a second bokeh server for diffraction and future apps which may cause similar problem. We make the change next week so I can update the website link.

**Comment by BenjaminRueth** [comment]
On the branch TestingServer I implemented the new functionality. The "TestingServer" is running on port 5100 (however this can be configured as usual using server.conf). In appnames_testing.conf the "problematic" apps are listed, currently we have Diffraction and Collision (see #8) here in order to prevent interference with the other apps.

I also updated the .exe files correspondingly.

As soon as you want to rollout the new server setup, just merge this branch into master.
